### PR TITLE
add long options

### PR DESCRIPTION
**Pull request by marchersimon**
I added long options for all the flags, since I think they are easier to remember. BTW, I'm also documenting the page in https://github.com/tldr-pages/tldr/pull/6098 right now, you could leave a comment if you want to.

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -47,11 +56,11 @@ int main(int argc, char *argv[]) {
 				break;
 			case 'h':
 				printf( "colorpicker [options]\n"
-						"  -h: show this help\n"
-						"  -o: oneshot\n"
-						"  -q: quit on keypress\n"
-						"  -d: hex only\n"
-						"  -r: rgb only\n");
+						"  -h, --help:             show this help\n"
+						"  -o, --one-shot:         oneshot\n"
```
Comment (bl-ue) [inline main.c:60]:
```suggestion
						"  -o, --oneshot:          oneshot\n"
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Comment by ym1234** [comment]
Thanks

**Comment by marchersimon** [comment]
No problem ;)
However there's a mistake @bl-ue pointed out which got merged. (The argument is `--oneshot` and I wrote `--one-shot` in the help part). Should I make a new PR for this or do you want to change it yourself?

**Comment by bl-ue** [comment]
#7 :)

**Comment by ym1234** [comment]
I like --one-shot more + oneshot isn't a word anyway.

**Comment by bl-ue** [comment]
@ym1234 I've updated that PR.  It's mergeable now.

**Pull request merged**